src/compas_occ/brep/brep.py

```diff
@@ -470,7 +486,7 @@ def from_shape(cls, shape: TopoDS_Shape) -> "BRep":
 
         """
         brep = cls()
-        brep.occ_shape = shape
+        brep.native_brep = shape
```

Comment:
perhaps we should add `sew` and `fix` here, and remove it everywhere else

Reply:
What do they do? 
If they're like Rhino's [`Brep.Repair()`](https://developer.rhino3d.com/api/rhinocommon/rhino.geometry.brep/repair) I'd advise against that. In Rhino it might actually create a weird invalid Brep if the input is not right.